Signal/src/Profiles/ProfileFetcherJob.swift

```diff
@@ -177,26 +175,26 @@ struct SignalServiceProfile {
             profileNameEncrypted = data
         }
 
-        var avatarUrlEncrypted: Data? = nil
+        var avatarUrlData: Data? = nil
         if let avatarUrlString = responseDict["avatar"] as? String {
```

Comment:
Is this really a base64 encoded string? I assumed the server was sending UTF-8 and the decoding we're doing in profile manager is unnecessary.